comp-layouts/input.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <script
+      src="https://kit.fontawesome.com/c4d0bd5372.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="/stylesheets/main.css">
+   <link rel="stylesheet" href="/stylesheets/components/input.css">
+  </head>
+  <body>
+    <div class="input-holder">
+      <label for="">Email</label>
+      <input type="text" class="input" placeholder="username@example.com" />
+    </div>
+
+    <div class="input-holder">
+      <label for="">Password</label>
+      <input type="password" class="input" placeholder="enter password" />
+      <p class="input-text">Password should be minimum 6 characters</p>
+    </div>
+
+    <div class="input-holder">
+      <label for="">Password</label>
+      <input type="text" class="input input-error" />
+      <p class="input-text input-text-error">No special characters allowed!</p>
+    </div>
+
+    <div class="input-holder">
+      <label for="">Email</label>
+      <div class="icon-chip">
+        <div class="input-holder-icon">
+          <i class="fas fa-envelope icon"></i>
+          <input type="text" class="input" placeholder="username@example.com" />
+        </div>
+      </div>
+    </div>
+
+    <div class="input-holder">
+      <label for="">Address</label>
+      <div class="icon-chip">
+        <div class="input-holder-icon">
```

Comment:
Naming convention can be improved here.`input-icon-holder` because description should be at the end.

Reply:
I have named it so , by keeping in mind , the hierarchical order of `.input-holder`.